src/local/butler/weights.py

```diff
@@ -102,16 +130,54 @@ def _list_fuzzer_jobs(fuzzer_jobs: Sequence[data_types.FuzzerJob]) -> None:
   for fuzzer_job in fuzzer_jobs:
     probability = fuzzer_job.actual_weight / total_weight
 
-    print("FuzzerJob:")
-    print(f'  Fuzzer: {fuzzer_job.fuzzer}')
-    print(f'  Job: {fuzzer_job.job}')
-    print(f'  Platform: {fuzzer_job.platform}')
-    print(f'  Weight: {fuzzer_job.actual_weight} = ' +
-          f'{fuzzer_job.weight} * {fuzzer_job.multiplier}')
-    print(f'  Probability: {_display_prob(probability)}')
+    printer("FuzzerJob:")
```

Comment:
nit: change this to a single quote please.

Reply:
Done.